LEAF_Request_Portal/sources/Telemetry.php

```diff
@@ -142,23 +143,15 @@ public function getRequestsPerMonth()
     }
 
     /**
-     * Purpose: Get total size of all uploads in portal folder
-     * @param $visn
-     * @param $facility
-     * @param $name
+     * Purpose: Get total size of all uploads in user upload directory
      * @return string
      */
-    public function getRequestUploadStorage() {
-
-        $command = 'du -sb ' . Config::$uploadDir;
-        $output = shell_exec($command);
-        if ($output) {
-            $sizeOutput = explode("\t", $output);
-            return $sizeOutput[0];
-
-        } else {
-            return '';
+    public function getRequestUploadStorage():String
```

Comment:
```suggestion
    public function getRequestUploadStorage():string
```

Should this be ":string"?